pkg/apis/seedmanagement/v1alpha1/types_managedseed.go

```diff
@@ -128,6 +128,8 @@ type GardenletDeployment struct {
 	// +optional
 	Env []corev1.EnvVar `json:"env,omitempty" protobuf:"bytes,10,rep,name=env"`
 	// VPA specifies whether to enable VPA for gardenlet. Defaults to true.
+	// Deprecated: This field is deprecated and has no effect anymore. It will be removed in the future.
+	// TODO(rfranzke): Remove this field after v1.110 has been released.
 	// +optional
 	VPA *bool `json:"vpa,omitempty" protobuf:"bytes,11,rep,name=vpa"`
```

Comment:
I would suggest to also clean up usages of this field from tests to avoid confusion. There are still usages of this field in unit and integration tests.